Application/EdFi.Ods.Api/Security/AuthorizationStrategies/NamespaceBased/NamespaceBasedAuthorizationFilterDefinitionsFactory.cs

```diff
@@ -61,10 +68,18 @@ public IReadOnlyList<AuthorizationFilterDefinition> CreateAuthorizationFilterDef
     private static void ApplyAuthorizationCriteria(
         ICriteria criteria,
         Junction @where,
-        string subjectEndpointName,
+        string[] subjectEndpointNames,
```

Comment:
Whereas joins for authorization have until now been on a single column, the custom view-based authorization opens us up to the possibility of composite key joins with multiple columns. Thus, the technical artifacts have been modified as needed to support this.